src/theme.ts

```diff
@@ -1,12 +1,131 @@
 import type { DefaultTheme } from 'styled-components';
 
+const FLEX_RECIPES = {
+  align: {
+    alignItems: 'center'
+  },
+  center: {
+    justifyContent: 'center',
+    alignItems: 'center'
+  }
+};
+
+const FLEX_VARIANTS = {
+  rowAlign: {
+    display: 'flex',
+    flexDirection: 'row',
+    ...FLEX_RECIPES.align
+  },
+  rowCenter: {
+    display: 'flex',
+    flexDirection: 'row',
+    ...FLEX_RECIPES.center
+  },
+  columnAlign: {
+    display: 'flex',
+    flexDirection: 'column',
+    ...FLEX_RECIPES.align
+  },
+  columnCenter: {
+    display: 'flex',
+    flexDirection: 'column',
+    ...FLEX_RECIPES.center
+  }
+};
+
 export const theme: DefaultTheme = {
```

Comment:
Since this theme will also be used on web apps it should also contain breakpoints

Reply:
Breakpoints might be very app specific though, right?

Reply:
It should be the same across all our products so the Components scale well between all of them :) Also a lot of those components will need a responsive version ;)

Reply:
Do you have any specific ones in mind?

Reply:
Core's breakpoints should do the job :)